lib/cn/csched_sp3.h

```diff
@@ -53,6 +53,16 @@ struct sp3_tree {
     struct list_head spt_dtree_linkv[2];
 };
 
+static inline void
+sp3_node_init(struct sp3_node *spn)
+{
+    for (uint tx = 0; tx < NELEM(spn->spn_rbe); ++tx)
```

Comment:
Should tx be a size_t? sizeof / sizeof = size_t?

Reply:
Done.